jni_gen/examples/pdfbox_plugin/dart_example/README.md

```diff
@@ -0,0 +1,10 @@
+## Running
+After generating `pdfbox_plugin` bindings in the parent directory
```

Comment:
I'm a bit confused about this directory structure. Why is `dart_example` a child of `pdfbox_plugin`, and not the other way around? If it's just because you have 2 examples that depend on it, `dart_example` and `example`, then you could just make the flutter `example` the parent, put `pdfbox_plugin` inside example, and just put `dart_example/bin/pdf_info.dart` the `example/bin` (since it's a standalone script).

Reply:
`dart_example` is a standalone dart project example without flutter dependency, `example` is a flutter GUI app example. There's also one more difference that one requires to run jni:setup and other does not. Should I merge them both?

Reply:
It is _not_ a Dart standalone script, because it requires dependencies and therefore a `pubspec.yaml`.

Merging means that the Dart example cannot be run anymore by users with just the Dart SDK without Flutter. So please unmerge them.